common/src/main/java/org/opencds/cqf/common/helpers/ClientHelper.java

```diff
@@ -0,0 +1,51 @@
+package org.opencds.cqf.common.helpers;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import ca.uhn.fhir.rest.client.api.ServerValidationModeEnum;
+import ca.uhn.fhir.rest.client.interceptor.BasicAuthInterceptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+
+@Component
+public class ClientHelper {
+
+    @PostConstruct
+    public void init() {
+        ClientHelper.environment = this.environmentTemp;
+    }
+
+    @Autowired
+    public Environment environmentTemp;
+
+    public static Environment environment;
```

Comment:
Let's have this be a plain ole class for now. We're not currently using the Spring features for this, and we're not using the environment injection.